clientcontroller/evm_consumer.go

```diff
@@ -81,34 +89,87 @@ func (ec *EVMConsumerController) QueryFinalityProviderVotingPower(fpPk *btcec.Pu
 }
 
 func (ec *EVMConsumerController) QueryLatestFinalizedBlocks(count uint64) ([]*types.BlockInfo, error) {
-	return ec.queryLatestBlocks(nil, count, finalitytypes.QueriedBlockStatus_FINALIZED, true)
+
+	var blocks []*types.BlockInfo
+
+	/*will be implemented in next PR, wait for "refactor: QueryLatestFinalizedBlocks(count uint64) -> QueryLatestFinalizedBlock() #347"
```

Comment:
ok it's already merged now. see the msg in TG group and update accordingly

you also need to sync changes from upstream and change base branch